lib/rules/color-hex-case/index.js

```diff
@@ -12,8 +12,19 @@ const messages = ruleMessages(ruleName, {
   expected: (actual, expected) => `Expected "${actual}" to be "${expected}"`,
 })
 
+const suggestFix = function (startIndex, hexValue, expectedHex) {
+  return {
+    insertion: expectedHex,
+    range: {
+      begin: startIndex,
+      end: startIndex + expectedHex.length,
```

Comment:
Should this be `hexValue.length`? I guess in this case it probably should not matter; but in other cases, we'll want to make sure we remove the whole original string when we replace it with another that might be of a different length.